backend/core/views.py

```
@@ -1916,6 +1970,10 @@
         context.update({"pk": self.kwargs["pk"]})
         return context
 
+
+class ComplianceAssessmentActionPlanList(ActionPlanList):
+    serializer_class = ComplianceAssessmentActionPlanSerializer
```

Ruff (0.11.9)

`ComplianceAssessmentActionPlanSerializer` may be undefined, or defined from star imports

(F405)